site/en/docs/devtools/css/flexbox/index.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: "layouts/doc-post.njk"
+title: "Examine and Modify CSS Flexbox Layouts"
+authors:
+  - jecelynyeen
+date: 2022-02-08
+#updated: YYYY-MM-DD
+description: "Learn how to use Chrome DevTools to examine and modify CSS Flexbox, as well as debugging layout issues."
+---
+
+{% YouTube id='J5n2aS37rpE' %}
+
+This guide shows you how to discover CSS flexbox on a page, editing them in the **Elements** panel of Chrome DevTools.
```

Comment:
suggestion: This guide shows you how to discover elements using CSS flexbox on a page and how to edit them in the **Elements** panel of Chrome DevTools.